source/reference/minio-mc/mc-mb.rst

```diff
@@ -22,6 +22,7 @@ specified path.
 
 You can also use :mc:`mc mb` against the local filesystem to produce
 similar results to the ``mkdir -p`` commandline tool.
+The S3 API allows for up to 500,000 buckets on a deployment or tenant.
```

Comment:
MinIO supports up to 500K buckets per deployment

since it's not an S3 limit, per se

Also you can do

```
.. cond:: k8s
   
   MinIO supports up to 500,000 buckets per tenant.

.. cond:: not k8s

  MinIO supports up to 500,000 buckets per deployment.
```

Reply:
👍